source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -270,23 +270,25 @@ export class Carousel extends Container {
 
         swiperContainer.appendChild(swiperWrapper as HTMLElement);
 
+        swiperContainer.tabIndex = 0;
+
         containerForAdorners.appendChild(swiperContainer);
 
         this.initializeCarouselControl(swiperContainer, nextElementDiv, prevElementDiv, pagination);
 
-        cardLevelContainer.addEventListener("focusin", (event) => {
-            if (!this._isSwiperInitialized) {
-                this._isSwiperInitialized = true;
-                this._swiper?.destroy();
-                this.initializeCarouselControl(swiperContainer, nextElementDiv, prevElementDiv, pagination);
-            }
-        });
+        cardLevelContainer.addEventListener("keydown", (event) => {
```

Comment:
why is the event changed for keydown? the issue can be hit if the controller is clicked on

Reply:
this is for an a11y config for pagination bullet since pages are not available at the point of execution, thus, pressing the spacebar on the bullet won't do anything. also key down events are narrower than focus in events.

Reply:
yeah, but if focusin handles clicking and keyboard events why handling keyboard only now?